src/logic/longPressContext.ts

```diff
@@ -0,0 +1,9 @@
+import React, { useContext } from 'react'
+
+export const LongPressContext = React.createContext<boolean>(false)
```

Comment:
Det går an å droppe `export` på denne, sidan me berre bruker dei to under i andre filer no.